src/RuleLocation.php

```diff
@@ -50,6 +50,32 @@ public function maxActionsCount()
         return 2;
     }
 
+    public function executeActions($output, $params, array $input = [])
+    {
+        if (count($this->actions)) {
```

Comment:
No need to embed the foreach in a `if (count($this->actions)) {`, the foreach will do nothing if array is empty.

Reply:
done